android/build.gradle

```diff
@@ -37,6 +36,7 @@ repositories {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.foursquare:pilgrimsdk:3.1.0'
-    implementation 'com.foursquare:pilgrimsdk-debugging:3.1.0'
+    implementation 'com.foursquare:pilgrimsdk:3.3.0'
+    implementation 'com.foursquare:pilgrimsdk-debugging:3.3.0'
+    implementation 'androidx.legacy:legacy-support-core-utils:1.0.0'
```

Comment:
@kylefowler i needed this other when starting/stopping Pilgrim I would get an error `Failed resolution of: Landroidx/legacy/content/WakefulBroadcastReceiver` is that something we need to fix?